### PR TITLE
Saves current edges overview view in query param

### PR DESCRIPTION
**Pull request by MTrost**

### CODE REVIEW

File: code/src/cljs/sixsq/nuvla/ui/edges/spec.cljs [hunk 1]
```diff
@@ -43,6 +43,15 @@
 
 (def default-ordering {:field :created :order "desc"})
 
+(def local-storage-key "nuvla.ui.edges-preferenes")
```
Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/edges/spec.cljs:46]: typo

File: code/src/cljs/sixsq/nuvla/ui/edges/events.cljs [hunk 2]
```diff
@@ -464,3 +480,23 @@
...
+;;
+(reg-event-fx
+  ::change-view-type
+  (fn [{{:keys [current-route]} :db} [_ new-view-type]]
+    (let [current-view  (keyword (-> current-route :query-params :view))
+          prefered-view {:view new-view-type}]
+      {:fx [(when (#{new-view-type current-view} spec/cluster-view)
+              [:dispatch [::pagination-plugin/change-page
+                          [::spec/pagination] 1]])
+            [:dispatch [::routing-events/change-query-param {:partial-query-params prefered-view}]]
```
Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/edges/events.cljs:493]: typo should be preferred

File: code/src/cljs/sixsq/nuvla/ui/edges/views.cljs [hunk 3]
```diff
@@ -178,34 +181,26 @@
...
-                                              [::spec/pagination] 1])
-                                   (reset! view-type :cluster))}
-         [ui/Icon {:className "fas fa-chart-network"}]]
+        (doall
+          (for [view spec/view-types]
+
+            ^{:key view}
+            [ui/MenuItem {:active   (= @view-type view)
+                          :on-click #(switch-view! view)}
+             [ui/Icon {:className (view->icon-classes view)}]]))
```
Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/edges/views.cljs:203]: Isn't you that pushed to replace with :class :P

Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/edges/views.cljs:203]: Isn't you that pushed to replace with :class :P

Comment (MTrost) [reply code/src/cljs/sixsq/nuvla/ui/edges/views.cljs:203]: I just use `:class` when I write it myself, but usually don't bother changing it. This one already existed so I didn't bother.

But now I'm curious how it came to be known that I pushed for this replacement 😄

**Pull request merged**